src/store/config.js

```diff
@@ -0,0 +1,12 @@
+import env from './modules/env'
+import starport from './modules/starport'
+import bank from './modules/cosmos/bank'
+import wallet from './modules/wallet'
+
+export default function init(store) {
+	store.registerModule(['modules'], { namespaced: true })
+	bank(store)
+	starport(store)
+	env(store)
+	wallet(store)
+}
```

Comment:
This is the only tricky-part of using the store. In order for the wiring-together of modules to work properly and allow ease-of-use inside components and one-line initialization, the _ordering_ here is **important** and because this config file is application-specific and written by the end-user developer, it is subject to errors.

- First line must be the registration of the "empty" modules namespace for further nesting to work.
- `env` and `wallet` store modules must register themselves at the end after all other stores modules have been initialized.

Reply:
`modules` name changed to `chain`